src/main/scala/group/scala/karazin/circe/literal/extras/macros.scala

```diff
@@ -161,9 +162,12 @@ object macros:
     
         case '[String] =>
           Json.fromString(StringUnit)
+
```

Comment:
It may be better to use one case for Short and Int.
```
case  '[Int] | '[Short] =>
    Json.fromInt(NumberUnit)
```

Reply:
> It may be better to use one case for Short and Int.
> 
> ```
> case  '[Int] | '[Short] =>
>     Json.fromInt(NumberUnit)
> ```

See my comment above.